create_aio_app/main.py

```diff
@@ -2,10 +2,7 @@
 from pathlib import Path
 
 import click
-from cookiecutter.exceptions import (
-    FailedHookException,
-    OutputDirExistsException
-)
+from cookiecutter.exceptions import FailedHookException, OutputDirExistsException
```

Comment:
i prefer use black only for created project not for package.

Reply:
reverted